jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/AsyncContextTest.java

```diff
@@ -71,7 +71,7 @@ public void setUp() throws Exception
         _contextHandler.addServlet(new ServletHolder(new TestServlet2()), "/servletPath2");
 
         ServletHolder testHolder = new ServletHolder(new TestServlet());
-        testHolder.setInitParameter("dispatchPath", "/test2/something%2felse");
+        testHolder.setInitParameter("dispatchPath", "/test2/something%25else");
```

Comment:
Ok to move from `%2F` to `%25`, but do we have `%2F` tests elsewhere now that these have been changed?
Now we throw if we find `%2F` (right?), but do we test that we actually throw?

Reply:
I've added another test for %2F, which currently dispatches with a request that returns the fully decoded pathInfo rather than throwing. I figure because it is coming from the app itself rather than over the wire, then we don't need to protect from ambiguous URIs because there is no security anyway.